### PR TITLE
Add webhosting.dk

### PR DESCRIPTION
**Pull request by SteffenGivard**
Adds [Webhosting.dk](https://www.webhosting.dk/?nl=ENG).

## About the difficulty

**TL;DR** I've set the difficulty to **hard**, as you'd have to wait 5 full years in order to be allowed to go trough with the deletion.

In theory, the deletion is simple, as you just need to visit the provided link and login to delete one's account. However, according to the company, due to a legal requirement to retain information for the last 5 financial years, user accounts cannot be deleted even if they are no longer in use. This retention is necessary because the accounts are linked to domain names, payment details with Nets, and invoices, as mandated by law.

In other words, one must wait 5 fulls years. 

The company are, in fact, _allowed_ (although not _permitted_, as they claim) to retain this info.
I'm extremely annoyed, that I cannot just issue a deletion, that'll take effect when my account meets their criteria. Instead, I'm supposed to remember to delete this in five years myself.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -19578,7 +19578,7 @@
     {
         "name": "Webex - Free Account",
         "url": "https://help.webex.com/en-us/5m4i4y/Delete-Your-Free-Webex-Account#id_111643",
-        "difficulty": "easy",
+        "difficulty": "medium",
         "notes": "Just follow the settings to find the delete account button and click it",
```
Comment (tupaschoal) [inline _data/sites.json:19582]: Is this change supposed to be here?

File: _data/sites.json [hunk 2]
```diff
@@ -19588,6 +19588,16 @@
         ]
     },
 
+    {
+        "name": "Webhosting.dk",
+        "url": "https://www.webhosting.dk/DKK/deleteaccount.php",
+        "difficulty": "easy",
+        "notes": "Due to a legal requirement to retain information for the last 5 financial years, user accounts cannot be deleted even if they are no longer in use. This retention is necessary because the accounts are linked to domain names, payment details with Nets, and invoices, as mandated by law.",
```
Comment (tupaschoal) [inline _data/sites.json:19595]: It's s tricky take, what happens if you try to login again, does it work? Or is the account purged immediate and just the data is kept for 5 years? If it's the latter, then I say we put it up as easy.

Lots of services can and do hold information, but many of them don't specify a period when it will be purged. In those cases we use impossible, but since this one claims the data is eventually erased, I think it's a different case.

File: _data/sites.json (hunk 1 above)
Comment (SteffenGivard) [reply _data/sites.json:19582]: Yes, it was :)

File: _data/sites.json (hunk 2 above)
Comment (SteffenGivard) [reply _data/sites.json:19595]: When trying to delete the account, a message about data needing to be stored for 5 years appears, and I cannot do anything but click "okay". The account isn't deleted, and I can log into it just fine. My guess is I'll have to wait for the 5 years to have passed, and then manually go in and re-request an account deletion. For that reason alone, I'd like to set the difficulty to "hard" 😅

Comment (tupaschoal) [reply _data/sites.json:19595]: I'd say you can even set it to impossible, since deleting an account is different from retaining the data, they could do the first without doing the latter

File: _data/sites.json (hunk 1 above)
Comment (tupaschoal) [reply _data/sites.json:19582]: But why? I'd expect a longer notes section if the difficulty turned to medium

Comment (SteffenGivard) [reply _data/sites.json:19582]: Oooh, I now see I've changes the difficulty on the wrong site. It was supposed to be for webhosting.dk. Sorry.

File: _data/sites.json (hunk 2 above)
Comment (SteffenGivard) [reply _data/sites.json]: It's not _impossible_, just really hard as one would have to wait 5 years, which is infinitely annoying, however it can be done 🤷

**Review by tupaschoal — APPROVED** [review]
